### ISSUE TITLE
uad doesn't work with screen readers at all

### ISSUE THREAD

**Issue by ahmedthebest31** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#415]
### Describe the bug

I am a blind user 
I try to use your software on Windows 
but it's appeared like a plank page 
my screen reader doesn't work with it at all. 

I'm using Windows 11 Version 23H2 
with NVDA screen reader version 2024.1
 I know creating graphical user interface in Rust not that easy but I hope if you can find a solution for this issue to make your software accessible for everyone. 
 thank you for your time and consideration.

### Expected behavior

I am a blind user 
I try to use your software on Windows 
but it's appeared like a plank page 
my screen reader doesn't work with it at all. 

I'm using Windows 11 Version 23H2 
with NVDA screen reader version 2024.1
 I know creating graphical user interface in Rust not that easy but I hope if you can find a solution for this issue to make your software accessible for everyone. 
 thank you for your time and consideration.

### You have a solution?

add support for screen readers like nvda on windows

### Provide logs

no log file needed

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by Frigyes06** [comment]
This is likely related to #49 and is a limitation of iced. Really annoying. I have no proposed solution for now.

**Comment by Rudxain** [comment]
#746 would be an alternative solution
